test/features/mailbox/presentation/utils/mapping_delete_multiple_mailbox.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/mail/mailbox/mailbox.dart';
+import 'package:model/mailbox/presentation_mailbox.dart';
+import 'package:tmail_ui_user/features/mailbox/presentation/model/mailbox_node.dart';
+import 'package:tmail_ui_user/features/mailbox/presentation/model/mailbox_tree.dart';
+import 'package:tmail_ui_user/features/mailbox/presentation/utils/mailbox_utils.dart';
+
+void main() {
+  group('mapping delete multiple mailbox test', () {
+
+    final expectedMapMailboxId = {
+      MailboxId(Id("A")): [
+        MailboxId(Id('A_1_1')),
+        MailboxId(Id('A_1')),
+        MailboxId(Id('A_2_1')),
+        MailboxId(Id('A_2')),
+        MailboxId(Id('A'))
+      ],
+      MailboxId(Id("B_1")): [MailboxId(Id('B_1'))],
+    };
+
+    test('_generateMapDescendantIdsAndMailboxIdList should return map mailbox id correct', () async {
```

Comment:
_generateMapDescendantIdsAndMailboxIdList should return map with 2 items when mailboxes belong to 2 different tree

Reply:
please add test case with 1 tree and 3 different tree also